### PR TITLE
Added setup instructions for the project

### PR DESCRIPTION
**Pull request by AlanApter**

### CODE REVIEW

**Comment by ibrahimsag** [comment]
For development: `rollup -w` should do the watch and rebuild next to `node local.js` for local serve. I will try to check if it needs adjustments.

**Review by xuelink — APPROVED** [review]
awesome

**Comment by AlanApter** [comment]
@ibrahimsag Thanks for the recommendation. I'm still getting familiar with JavaScript development, so your suggestion is very helpful. I've incorporated the changes into this pull request.

File: readme.md [hunk 1]
```diff
@@ -33,3 +33,76 @@ Screens are part of the computers for about fifty years now. But couple thousand
...
+   node local.js
+   ```
+
+   Upon successful startup, the command line will display a message similar to:
+
+   ```
+   listening at http://localhost:<port number>
+   ```
+
+   Replace `<port number>` with the actual port number displayed, such as `3000`.
```
Comment (ibrahimsag) [inline readme.md]: this option doesn't really exist.

File: readme.md [hunk 2]
```diff
@@ -33,3 +33,76 @@ Screens are part of the computers for about fifty years now. But couple thousand
...
+   For ongoing development, use Rollup in watch mode to automatically rebuild files when changes are detected. Start Rollup in watch mode for both configurations in separate CLI windows or tabs:
+
+   * For core project files:
+   ```
+   rollup -c rollup.min.elements.config.js -w
+   ```
+   * For project elements:
+   ```
+   rollup -c rollup.min.config.js -w
+   ```
```
Comment (ibrahimsag) [inline readme.md:89]: `rollup.config.js` should cover for both during development. And I think it is the default option if not provided as an explicit argument.

**Review by ibrahimsag — COMMENTED** [review]
Thanks for taking the time to document these steps. Let me know if you need further help.

File: readme.md (hunk 1 above)
Comment (AlanApter) [reply readme.md]: Done

File: readme.md (hunk 2 above)
Comment (AlanApter) [reply readme.md:89]: Done

**Pull request merged**